### ISSUE TITLE
Preferences window name "Preferences" is hardly visible in Black mode of High contrast.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#341]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.

1. Launch Xamarin Workbooks application.Which invokes "Create New Workbook" window.
2. Navigate to Tools and Preferences. Invoke preferences.
3. Verify the Preferences window name in black mode of High contrast.

### Expected Behavior
Window name should be clearly visible.
### Actual Behavior
"Preferences" text is hardly visible in High contrast black mode.

Note: In high contrast white mode "Preferences" window name is clearly visible

### User Impact
Low vision user will feel difficult to see the "Preferences text"

### Screenshots
![6 clearly visible hc white](https://user-images.githubusercontent.com/22956130/40722699-a18e437c-643a-11e8-8733-1649e7d20033.PNG)
![6 normal mode](https://user-images.githubusercontent.com/22956130/40722700-a1b9b958-643a-11e8-810c-f02b3b41f2f9.PNG)
![6 preferences window hardly visible hcb](https://user-images.githubusercontent.com/22956130/40722702-a1e24b7a-643a-11e8-8547-662e9bda34c1.PNG)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
